cli.js

```diff
@@ -78,6 +87,6 @@ process.stdin.on('keypress', (ch, key) => {
 		}
 
 		prevResult = emojis;
-		logUpdate(`${pre}${chalk.bold(query.join(''))}\n${emojis}`);
+		logUpdate(`${pre}${chalk.bold(query.join(''))}\n${emojis}\n`);
```

Comment:
That last `\n` is used __after__ the emojis are displayed, as if someone hit `ctrl + c` then it should quit by providing equal space. Also, it was looking like a mess, so I thought of putting this into the tool.

![screenshot from 2016-06-23 18 16 39](https://cloud.githubusercontent.com/assets/9411252/16303578/01411bc6-396f-11e6-887e-60c804ebb4ab.png)